ras/ras_extended.py

```diff
@@ -150,9 +150,9 @@ def test1_bootlist(self):
             cmd = "bootlist %s" % list_item
             self.run_cmd(cmd)
         interface = self.run_cmd_out(
-            "ifconfig | head -1 | cut -d':' -f1")
-        disk_name = self.run_cmd_out("df -h | egrep '(s|v)d[a-z][1-8]' | "
-                                     "tail -1 | cut -d' ' -f1").strip("12345")
+            "ifconfig | egrep '^(net|eth)' | cut -d':' -f1")
```

Comment:
@naveen538 Since we are already here can you change the functionality in python?